Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -314,23 +324,46 @@ void OpenALStream::SoundLoop()
     // many silence samples.  These do not need to be timestretched.
     if (rate > 0.10)
     {
-      soundTouch.setTempo(rate);
-      if (rate > 10)
+      if (use_timestretching)
+      {
+        soundTouch.setTempo(rate);
+      }
+      else
+      {
+        alSourcef(ui_source, AL_PITCH, static_cast<ALfloat>(rate));
```

Comment:
Here they are:
Resampled using OpenAL:
https://youtu.be/3ldr1mumVdA

Resampled using Dolphin:
https://youtu.be/dBkP0nFBd9g

Time-stretching using SoundTouch:
https://youtu.be/0PVNWL3oPUQ

I also attached here an Audacity image showing what happens to the audio when resampled using Dolphin.
![captura2r](https://cloud.githubusercontent.com/assets/14236067/24731020/d8187b76-1a3d-11e7-9863-9ed63417505a.PNG)